src/main/python/main.py

```diff
@@ -0,0 +1,26 @@
+from single_jointed_arm import single_jointed_arm
+from arm_sim import animate_arm_trajectory
+from math import *
+from math_util import RK4
+from arm_control_law import P
+
```

Comment:
Although I wouldn't say it's required now, please set up the python package.

Reply:
Also, since the sim isn't part of the actual robot code, maybe move the `python` folder to the root of the repo?

Reply:
I'm fine with that, as long as there is something listing the dependencies.